WW_FullyLeptonic/WW_FullyLeptonic.lua

```diff
@@ -0,0 +1,136 @@
+    ps_point = getpspoint(),
+    reco_particle = 'input::particles/1',
+    sigma = 0.05,
+}
+
+GaussianTransferFunction.tf_p2 = {
+    ps_point = getpspoint(),
+    reco_particle = 'input::particles/2',
+    sigma = 0.10,
+}
+
+inputs_before_perm = {
+    'tf_p1::output',
+    'tf_p2::output',
+}
+
+
+inputs = inputs_before_perm
+
+-- Use the BreitWignerGenerators to generate values distributed as the corresponding peaks,
+-- for each propagator in the topology
+BreitWignerGenerator.flatter_s13 = {
+    ps_point = getpspoint(),
+    mass = parameter('W_mass'),
+    width = parameter('W_width')
+}
+
+BreitWignerGenerator.flatter_s24 = {
+    ps_point = getpspoint(),
+    mass = parameter('W_mass'),
+    width = parameter('W_width')
+}
+
+-- The main block defines the phase-space parametrisation,
+-- converts our particles given by the transfer functions, and our propagator masses
+-- into solutions for the missing particles in the event
+BlockF.blockf = {
+    inputs = inputs,
+
+    -- Fix the neutrino transverse momentum to the experimental MET passed to MoMEMta
+    pT_is_met = true,
+
+    s13 = 'flatter_s13::s',
+    s24 = 'flatter_s24::s',
+    q1 = getpspoint(),
+    q2 = getpspoint()
+}
+
+-- Using the fully reconstructed event (invisibles and visibles), build the initial state
+BuildInitialState.boost = {
+    particles = inputs,
+    invisibles = {
+        'blockf::invisibles',
+    },
+
+    -- Since the neutrinos were reconstructed using the experimental MET,
+    -- the event has non-zero total transverse momentum,
+    -- so we have to do the boost to define an initial state satisfying conservation
+    -- of momentum.
+    do_transverse_boost = true
+}
+
+jacobians = {'flatter_s13::jacobian', 'flatter_s24::jacobian', 'tf_p1::TF_times_jacobian', 'tf_p2::TF_times_jacobian'}
+
+-- This module defines the `integrands` output, which will be taken by MoMEMta as the value to integrate
+MatrixElement.WW = {
+  pdf = 'CT10nlo',
+  pdf_scale = parameter('W_mass'),
+
+  -- Name of the matrix element, defined in the .cc file of the ME 
+  matrix_element = 'pp_WW_fully_leptonic',
+  matrix_element_parameters = {
+      card = '../WW_FullyLeptonic/pp_WW_fully_leptonic/Cards/param_card.dat'
+  },
+
+  initialState = 'boost::output',
+
+  invisibles = {
+    input = 'blockf::invisibles',
+    -- Jacobians of the block: one value per invisibles' solution
+    jacobians = 'blockf::jacobians',
+    ids = {
+      {
+        pdg_id = 12,
+        me_index = 2,
+      },
+
+      {
+        pdg_id = -14,
+        me_index = 4,
+      }
+    }
+  },
+
+  -- Configure how the inputs are linked to the matrix element (order and PID of the leg)
+  -- Together with the invisibles' ids, it has to match the `mapFinalStates` index in the matrix element .cc file
+  particles = {
+    inputs = inputs,
+    ids = {
+      {
+        pdg_id = -11,
+        me_index = 1,
+      },
+
+      {
+        pdg_id = 13,
```

Comment:
Looking at the particle IDs, it seems you have WW > e mu, but in the first line the matrix element is called "WW_ee". This is a bit confusing...